packages/server/src/course/course.controller.ts

```diff
@@ -370,6 +371,7 @@ export class CourseController {
       {
         room,
         courseId,
+        isDisabled: false,
```

Comment:
I wonder if it's also worth adding a `disabled` check to `QueueModel.checkIsOpen()` - it looks like the `createQuestion` endpoint doesn't explicitly check that the question is being created on a disabled queue, but it does do `checkIsOpen`. `checkIsOpen` just checks if the queue has any staff, and theoretically a disabled queue should never have any staff - but it might be nice to also make sure that it's also not disabled to be safe.